docs/spec.md

```diff
@@ -1530,22 +1548,26 @@ strict lower triangle correspondingly, are implementation-defined.
 If there exists `i` where the input matrix is not an Hermitian positive-definite
 matrix, then the behavior is undefined.
 
+For quantized types, performs
+`dequantize_op_quantize(lambda operand: cholesky(operand, lower), a, type(result))`.
+
 #### Inputs
 
 | Label | Name    | Type                                       | Constraints |
 |-------|---------|--------------------------------------------|-------------|
-| (I1)  | `a`     | tensor of floating-point or complex type   | (C1-C3)     |
+| (I1)  | `a`     | tensor of floating-point or complex type or per-tensor
+quantized tensor   | (C1-C3)     |
```

Comment:
I think we'll also want to align other rows of this table with the updated length of this row.

Reply:
done.